helm/kanister-operator/templates/deployment.yaml

```
@@ -38,4 +38,14 @@ spec:
 {{- if .Values.resources }}
         resources:
 {{ toYaml .Values.resources | indent 12 }}
+{{- end }}
+      - name: {{ template "repository-server-controller.name" . }}
+        image: {{ .Values.repositoryServerImage.repository }}:{{ .Values.repositoryServerImage.tag }}
```

```suggestion
        image: {{ .Values.repositoryServerImage.registry }}/{{ .Values.repositoryServerImage.name }}:{{ .Values.repositoryServerImage.tag }}
```
If we split image into registry and name, this would be required.